modules/sdk-core/src/account-lib/mpc/shamir.ts

```diff
@@ -27,7 +27,7 @@ export default class Shamir {
     numShares: number,
     indices?: Array<number>,
     salt = BigInt(0)
-  ): Record<number, bigint> {
+  ): [Record<number, bigint>, Array<bigint>] {
```

Comment:
NIT: Can we update the new return type in the return type description of this function also. Also if it makes sense, instead of returning shares and `v` as an array, can we consider to return it as an object like 
```suggestion
  ): { shares: Record<number, bigint>,v:  Array<bigint>} {
```
Would be become more clearer on what exactly is being returned.